govcd/lb_test.go

```diff
@@ -18,7 +18,7 @@ import (
 	. "gopkg.in/check.v1"
 )
 
-// Test_LB load balancer integration test
+// Test_Lb load balancer integration test
 // MANDATORY PREREQUISITE - manually enable load balancing for edge gateway with enabled acceleration
```

Comment:
Perhaps we can remove the "manually enable", since the test now does it

Reply:
Done.